nvidia/daemonset.yaml

```diff
@@ -17,12 +17,12 @@ spec:
     spec:
       hostPID: true
       initContainers:
-      - image: squat/modulus:4c946362b8267c4680618c39489d873ed386f807
+      - image: squat/modulus:6af521a148f51e164764de716da0b30769c57083
```

Comment:
Should we bump this now that #22 is merged?

Reply:
```suggestion
      - image: squat/modulus:85f9128a80cf47adea85fc008b8c2dcbfad6da86
```

Reply:
@squat Good shout, done!